packages/ember-htmlbars/lib/helper.js

```diff
@@ -41,6 +41,33 @@ import Object from 'ember-runtime/system/object';
 
   Additionally, class helpers can call `recompute` to force a new computation.
 
+  If you only require a simple functional style helper (where the output is only
```

Comment:
This could be shortened to "If you require a helper where the output is only dependent on the current input, then you can use the `Helper.helper` function".